.github/workflows/anchor-test.yml

```
@@ -7,25 +7,123 @@ on:
+
+      - name: Cache Solana CLI tools
+        uses: actions/cache@v4
+        with:
+          path: |
+            ~/.cache/solana/
+            ~/.local/share/solana/
+            ~/.config/solana/
+          key: solana-cli-${{ runner.os }}-build-${{ env.SOLANA_VERSION }}
+
+      - name: Install Solana CLI tools
+        if: steps.cache-solana.outputs.cache-hit != 'true'
```

actionlint (1.7.4)

property "cache-solana" is not defined in object type {}

(expression)